src/BenchmarkDotNet/Portability/EffectiveFrequency/MacEffectiveCpuProvider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using BenchmarkDotNet.Helpers;
+using BenchmarkDotNet.Portability.Cpu;
+
+namespace BenchmarkDotNet.Portability.EffectiveFrequency
+{
+    internal class MacEffectiveCpuProvider
+    {
+        internal static readonly Lazy<CpuInfo> MacEffectiveCpuInfo = new Lazy<CpuInfo>(GetInfo);
+        
+        private static CpuInfo GetInfo()
+        {
+            
+            if (RuntimeInformation.IsMacOSX())
+            {
+                string content = ProcessHelper.RunAndReadOutput("sysctl", "-a");
```

Comment:
We already call `sysctl -a` in `SysctlCpuInfoProvider`. Please, reuse the result instead of another call.